tests/test_trans_linearize.py

```diff
@@ -176,6 +176,30 @@ def test_impl(self):
         cons = linearize_constraint(cons)[0]
         self.assertEqual("(~bv) -> (sum([1, 2, 3] * [a, b, c]) <= 15)", str(cons))
 
+    def test_mod(self):
+
+        x,y = cp.intvar(1,5, name="x"), cp.intvar(1,5,name="y")
+        rhs = cp.intvar(-5,5, name="rhs")
+
+        cons = (x % y) <= rhs
+        sols = set()
+        cp.Model(cons).solveAll(display=lambda : sols.add((x.value(), y.value(), rhs.value())))
+        lincons = linearize_constraint([cons], supported={"sum", "wsum", "mod"})
```

Comment:
Are you sure this tests what it should? "mod" in supported is not used anywhere, what was added uses "mul" in supported.